apps/mobile/src/libs/constants/gradient-sectors-data.constant.ts

```diff
@@ -0,0 +1,15 @@
+import { GradientColor } from "../enums/enums";
+import { type WheelDataItem } from "../types/types";
+
+const GRADIENT_SECTORS_DATA: WheelDataItem[] = [
```

Comment:
```suggestion
const GRADIENT_SECTORS_INITIAL_DATA: WheelDataItem[] = [
```
Since it's not real data